File: LeetCode/242-Valid-Anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {

        if(s.Length != t.Length)
            return false;
            
        char[] sChar = s.ToCharArray();
        char[] tChar = t.ToCharArray();

        Array.Sort(sChar);
        Array.Sort(tChar);

        return new string(sChar) == new string(tChar); 

        
    }
}

File: LeetCode/242-Valid-Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {

        if (s.length() != t.length())
        return false;

        std::string sSorted = s;
        std::string tSorted = t;

        std::sort(sSorted.begin(), sSorted.end());
        std::sort(tSorted.begin(), tSorted.end());

        return sSorted == tSorted;
    }
};